File: web3rpc/rpc-specs/code-samples/python/eth/miscellaneous/submitWork.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

nonce = "0x0000000000000001"
powHash = "0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef"
mixDigest = "0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.submit_work(nonce, powHash, mixDigest)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/miscellaneous/submitWork.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const nonce = '0x0000000000000001'
    const powHash = '0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef'
    const mixDigest = '0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef'
    
    provider.eth.submitWork(nonce, powHash, mixDigest, {}, (err, data, response) => {
        console.log(data);
    });
}
)()